src/features/admin/evaluation-administrations/[id]/progress/evaluation-progress-list.tsx

```diff
@@ -44,6 +45,8 @@ export const EvaluationProgressList = () => {
   const [showEmailLogDialog, setShowEmailLogDialog] = useState<boolean>(false)
   const [selectedEvaluationId, setSelectedEvaluationId] = useState<number | null>(null)
 
+  // const NAME_LENGTH = 10
```

Comment:
I think this should be removed?

Reply:
Yeapp2x I forgot to remove the constant my bad. :3